WalletWasabi.Fluent/ViewModels/Wallets/Advanced/WalletCoins/WalletCoinsViewModel.cs

```diff
@@ -24,40 +24,40 @@ public partial class WalletCoinsViewModel : RoutableViewModel
 {
 	private readonly IObservable<Unit> _balanceChanged;
 	private readonly WalletViewModel _walletViewModel;
-	private ObservableAsPropertyHelper<bool> _isAnySelected = ObservableAsPropertyHelper<bool>.Default();
-	[AutoNotify] private FlatTreeDataGridSource<WalletCoinViewModel> _source = new(Enumerable.Empty<WalletCoinViewModel>());
+	[AutoNotify] private IObservable<bool> _isAnySelected = Observable.Return(false);
+
+	[AutoNotify]
+	private FlatTreeDataGridSource<WalletCoinViewModel> _source = new(Enumerable.Empty<WalletCoinViewModel>());
 
 	public WalletCoinsViewModel(WalletViewModel walletViewModel, IObservable<Unit> balanceChanged)
 	{
 		_walletViewModel = walletViewModel;
 		_balanceChanged = balanceChanged;
-        SetupCancel(enableCancel: false, enableCancelOnEscape: true, enableCancelOnPressed: true);
+		SetupCancel(false, true, true);
```

Comment:
It seems there is something in your style format configuration that always removes the parameter names.

Reply:
🤦‍♂️ReSharper... I'll review the config, because it's a recurrent problem...

Reply:
OK; it seems that it does this by default. I've created a different configuration for Wasabi Wallet with this option disabled:
![image](https://user-images.githubusercontent.com/3109851/183389375-bd267359-e7dc-48a4-8f4b-423c9a63d8df.png)